### ISSUE TITLE
Space used by user isn't precise

### ISSUE THREAD

**Issue by yurivict** [HelloZeroNet/ZeroBlog#25]
For some reason, the same count shows differently on different peers/browers:
USED: 3.0K/15K
USED: 2.5k/15k
USED: 2.9k/15k

Messages are displayed exactly the same way, but sizes differ, which is weird.

**Comment by HelloZeroNet** [comment]
this displays the space used by the current logged in user and it also includes the current comment you are writinig

**Comment by yurivict** [comment]
Yes, it shows different numbers for the same user and the same set of posts.
